### PR TITLE
feat(route): add github all issue/pull comments

### PR DESCRIPTION
**Pull request by FliegendeWurst**
## 该 PR 相关 Issue / Involved Issue

None

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/github/comments/DIYgod/RSSHub
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route (modified)
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/github/comments.js [hunk 1]
```diff
@@ -17,8 +21,8 @@ const typeDict = {
 module.exports = async (ctx) => {
     const user = ctx.params.user;
     const repo = ctx.params.repo;
-    const number = isNaN(parseInt(ctx.params.number)) ? 1 : parseInt(ctx.params.number);
-    const limit = ctx.query.limit ? parseInt(ctx.params.limit) : 100;
+    const number = ctx.params.number && isNaN(parseInt(ctx.params.number)) ? 1 : parseInt(ctx.params.number);
+    const limit = ctx.params.limit ? parseInt(ctx.params.limit) : 100;
```
Comment (TonyRL) [inline lib/v2/github/comments.js:25]:
```suggestion
    const limit = ctx.query.limit ? parseInt(ctx.query.limit) : 100;
```
That was my typo.

File: lib/v2/github/comments.js [hunk 2]
```diff
@@ -29,6 +33,65 @@ module.exports = async (ctx) => {
...
+        const issueType = typeDict[urlParts[urlParts.length - 2]].title;
+
+        items.push({
+            title: `${actor} commented on ${user}/${repo}: ${issueType} #${issue}`,
+            author: actor,
+            pubDate: parseDate(item.created_at),
+            link: item.html_url,
+            description: item.body,
+        });
+    });
```
Comment (TonyRL) [inline lib/v2/github/comments.js:70]: Please use [.map()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) instead.

File: lib/v2/github/comments.js [hunk 3]
```diff
@@ -29,6 +33,65 @@ module.exports = async (ctx) => {
...
+        const issue = issueUrlParts[issueUrlParts.length - 1];
+        const urlParts = item.html_url.split("/");
+        const issueType = typeDict[urlParts[urlParts.length - 2]].title;
+
+        items.push({
+            title: `${actor} commented on ${user}/${repo}: ${issueType} #${issue}`,
+            author: actor,
+            pubDate: parseDate(item.created_at),
+            link: item.html_url,
+            description: item.body,
```
Comment (TonyRL) [inline lib/v2/github/comments.js:68]: Please render markdown content.

**Comment by FliegendeWurst** [comment]
Thanks for the quick review @TonyRL. I have addressed all of your points.

**Pull request merged**